src/aux-server/aux-web/aux-player/PlayerApp/PlayerApp.ts

```diff
@@ -1247,6 +1248,30 @@ export default class PlayerApp extends Vue {
                     this._watchLoom(e, simulation);
                 } else if (e.type === 'get_loom_metadata') {
                     this._getLoomMetadata(e, simulation);
+                } else if (e.type === 'get_script_issues') {
+                    this._getScriptIssues(e, simulation);
+                    try {
+                        const helpers = await import(
+                            '../../shared/MonacoHelpers'
+                        );
+                        const bot = simulation.helper.botsState[e.botId];
+                        const issues = await helpers.getScriptIssues(
+                            simulation,
+                            bot,
+                            e.tag
+                        );
+
+                        simulation.helper.transaction(
+                            asyncResult(e.taskId, issues)
+                        );
+                    } catch (ex) {
+                        if (hasValue(e.taskId)) {
+                            simulation.helper.transaction(
+                                asyncError(e.taskId, ex.toString())
+                            );
+                            console.log('Error fetching issues:', ex);
+                        }
+                    }
```

Comment:
@TroyceGowdy This try/catch block appears to duplicate what the `_getScriptIssues()` function does. Can you refactor the code so that it does everything it needs inside the `_getScriptIssues()` function?